client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -983,6 +997,43 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: pluginsIcon,
 		},
+		{
+			name: 'changePlan',
+			label: __( 'Change plan on a site' ),
+			searchLabel: [
+				_x( 'upgrade plan', 'Keyword for the Change plan on your site command' ),
+				_x( 'change plan', 'Keyword for the Change plan on your site command' ),
+				_x( 'add plan', 'Keyword for the Change plan on your site command' ),
+			].join( ' ' ),
+			context: [ '/sites' ],
+			callback: setStateCallback( 'changePlan', __( 'Select site to change plan' ) ),
+			siteFunctions: {
+				onClick: ( { site, close }: { site: SiteExcerptData; close: () => void } ) => {
+					close();
+					navigate( `/plans/${ site.slug }` );
+				},
+				filter: ( site: SiteExcerptData ) => ! isP2Site( site ) && ! site?.is_wpcom_staging_site,
+			},
+			icon: creditCardIcon,
+		},
+		{
+			name: 'manageMyPlan',
+			label: __( 'Manage plan on a site' ),
```

Comment:
```suggestion
			label: __( 'Manage site plan' ),
```

Reply:
Changed in https://github.com/Automattic/wp-calypso/pull/85083/commits/47e470467aa9010a7f2697c44147d98d69aef2d4